Kernel/Arch/aarch64/vector_table.S

```diff
@@ -208,5 +208,7 @@ system_error_lower_el:
 
 .global restore_context_and_eret
 restore_context_and_eret:
+    mov x0, sp
+    bl exit_trap
```

Comment:
Ah, nice find! That was one of the FIXME's I had laying around in my local patches. (This is just a comment)